packages/eds-core-react/src/components/Button/ButtonGroup/ButtonGroup.docs.mdx

```diff
@@ -0,0 +1,60 @@
+import { Story, PropsTable, Links } from './../../../../.storybook/components'
+import { Button } from './..'
+
+# Group
+
+Is used to **group** related Buttons.
+
+<Links 
+ariaUrl="https://w3c.github.io/aria/#group" 
+sourceUrl="https://github.com/equinor/design-system/blob/develop/packages/eds-core-react/src/components/Button/ButtonGroup"
+npmUrl="https://www.npmjs.com/package/@equinor/eds-core-react/"
+/>
+
+<Story id="inputs-buttons-group--introduction" />
+<PropsTable />
+
+## Usage
+
+The buttons can be grouped by wrapping them with the `Button.Group` component.
+
+```tsx
+import { Button } from '@equinor/eds-core-react'
+
+<Button.Group aria-label="primary">
+  <Button>Click me!</Button>
+  <Button>Click me!</Button>
+  <Button>Click me!</Button>
+</Button.Group>
+```
+
+## Accessibility
+
+### ARIA
+<ul>
+  <li>Button.Group has <code>role="group"</code>. The word <code>group</code> will be read by assistive technologies.</li>
+  <li>You should provide an accessible label with <code>aria-label="label"</code>, <code>aria-labelledby="id"</code> or <code>&lt;label&gt;</code>.</li>
+</ul>
+
+## Examples
+
+### Horizontal
+The `Button.Group` is displayed horizontally by default.
+
+<Story id="inputs-buttons-group--horizontal" />
+
+### Vertical
+The `Button.Group` can be displayed vertically using the `vertical` prop.
```

Comment:
Suggest merging these into one story and just calling it "orientation"? Then just have both the horizontal and vertical in the same story.